app/views/users/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Here is a list of users</h1>
-<p>Find me in app/views/user/index.html.erb</p>
+<div class="users_index_main_container" >
+<% if @users %>
+  <% @users.each do |user| %>
+    <div class="user_container" >
+      <div class="user_photo_container" >
+        <img src='<%= user.photo %>' a='user profile picture'>
+      </div>
+      <div class="user_profile_container" >
+        <p class="user_name" ><%= user.name %></p>
+        <p class="user_post_counter">Number of posts: <%= user.posts_counter %></p>
+      </div>
+    </div>
```

Comment:
**Changes Required**
- [ ] Clicking on each user takes you to their show page. Please let us add a link to the user show page when one clicks on user as per the requirement.